examples/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart

```diff
@@ -166,30 +178,38 @@ class _BottomAppBarDemoState extends State<BottomAppBarDemo> {
     ];
     for (Color color in babColors) {
       colors.add(
-        new Radio<Color>(
-          value: color,
-          groupValue: babColor,
-          onChanged: (Color color) {
-            setState(() {
-              babColor = color;
-            });
-          },
-        ),
-      );
-      colors.add(
-        new Container(
-          decoration: new BoxDecoration(
-            color: color,
-            border: new Border.all(width:2.0, color: Colors.black),
-          ),
-          child: const SizedBox(width: 20.0, height: 20.0),
+        new Semantics(
+          label: 'Set Bottom App Bar color to ${colorToName[color]}',
+          container: true,
+          explicitChildNodes: false,
```

Comment:
nit: `explicitChildNodes: false` is default behavior and should probably just be removed.

Reply:
I've gone ahead and done that.

I merged this before seeing this comment from mobile.  I'm too used to our internal UI where any comments blocks submitting a change. 

I'll send a PR with this update and remember to check for final comments before merging next time.